samples/highcharts/blog/read-csv-data-using-beforeparse/demo.js

```diff
@@ -0,0 +1,42 @@
+Highcharts.setOptions({
+    colors: ["#058DC7", "#c7058d"]
+});
```

Comment:
You can move the `colors` array directly into the chart's options (you don't need to use Highcharts.setOptions())